File: pinyougou_parent/pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SeckillService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {

    /**
     * 从缓存中获取秒杀商品列表展示
     */
    public List<TbSeckillGoods> findSeckillGoodsListFromRedis();

    TbSeckillGoods findOneSeckillGoods(Long seckillGoodsId);

    /**
     * 保存秒杀订单
     */
    public void createSeckillOrder(Long seckillGoodsId,String userId);
}

File: pinyougou_parent/pinyougou_seckill_web/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//获取秒杀商品列表展示
    this.findSeckillGoodsList=function () {
        return $http.get("seckill/findSeckillGoodsList.do");
    }
    //查询秒杀商品详情信息
    this.findOneSeckillGoods=function (seckillGoodsId) {
        return $http.get("seckill/findOneSeckillGoods.do?seckillGoodsId="+seckillGoodsId);
    }

    //秒杀下单
    this.createSeckillOrder=function (seckillGoodsId) {
        return $http.get("seckill/createSeckillOrder.do?seckillGoodsId="+seckillGoodsId);
    }

});
